azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -212,6 +220,18 @@ public KubernetesContainerizedImpl(final Props azkProps,
         this.azkProps
             .getString(ContainerizedDispatchManagerProperties.KUBERNETES_MOUNT_PATH_FOR_JOBTYPES,
                 DEFAULT_APP_MOUNT_PATH_PREFIX_FOR_JOBTYPES);
+    this.dependencySet =
```

Comment:
I believe it will be up to us to define the naming conventions for any `dependencies`, should consider defining a consistent naming format to avoid any issues/confusion similar to the ones we have seen for jobType names.

Reply:
changed to dependencyTypes